src/lib/button-area/button-area.scss

```diff
@@ -1,11 +1,62 @@
-@use './mixins';
+@use './core' as *;
+@use '../state-layer';
+@use '../focus-indicator';
+@use '../utils/mixins' as utils;
 
-@include mixins.core-styles;
+//
+// Host
+//
+
+$host-tokens: [disabled-cursor focus-indicator-color];
```

Comment:
Do these _need_ to be host-level tokens? I think you can just keep them defined within the shadow root.